packages/core/package.json

```diff
@@ -23,7 +23,9 @@
     "webpack": "3.8.1"
   },
   "description": "Wire for Web's communication core.",
-  "files": ["dist/commonjs"],
+  "files": [
+    "dist/commonjs"
+  ],
```

Comment:
We definitely need to align our setups. This is going back and forth again and again.

Reply:
@dependabot is working on it.. so it will be fixed :)